### PR TITLE
Rhmap 8262 request id logging

### PR DESCRIPTION
**Pull request by nialldonnellyfh**

### CODE REVIEW

File: lib/request-id/decorateLogger.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/*
+ JBoss, Home of Professional Open Source
+ Copyright Red Hat, Inc., and individual contributors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (fh-qube) [inline lib/request-id/decorateLogger.js]: MAJOR Remove this commented out code. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)

File: lib/request-id/middleware.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/*
+ JBoss, Home of Professional Open Source
+ Copyright Red Hat, Inc., and individual contributors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (fh-qube) [inline lib/request-id/middleware.js]: MAJOR Remove this commented out code. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)

**Comment by fh-qube** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:
- The code changes conform to the contributing & code style guides for this component (if they exist)
- A component owner should review your change (we've tagged these above)
- Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
- If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
- Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
- Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by fh-qube** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
- MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
- MAJOR [Remove this commented out code.](https://github.com/feedhenry/fh-logger/blob/da25dd1be37d3c4c59f51879f978ea0dcbf383a4/lib/request-id/middleware.js#L5) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)

**Pull request merged**